src/styles.css

```diff
@@ -0,0 +1,115 @@
+/*  Add your styling here */
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: sans-serif;
+}
+.logo {
+  height:80px;
+  
+}
+
+header h2 {
+  line-height: 80px;
+  font-size: 24px;
+  text-transform: uppercase;
+}
+
+.header-nav {
+  background-color: red;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  width:100%;
+  
+}
+.check-btn {
+  font-size: 50px;
+  color: white;
+  cursor: pointer;
+}
+.menubtn-text {
+  font-size: 15px;
+  text-transform: uppercase;
+  color: black;
+  font-weight: bold;
+}
+
+.nav-bar {
+  top: 100px;
+  position:fixed;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items:center;
+  transition: all 0.5s;
```

Comment:
good job on using transitions, it makes your menu look very smooth when clicking through 👏